gapic/templates/setup.py.j2

```diff
@@ -33,10 +33,16 @@ dependencies = [
     "protobuf>=3.19.5,<5.0.0dev,!=3.20.0,!=3.20.1,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5",
     {# TODO: Remove after https://github.com/googleapis/gapic-generator-python/pull/1240 is merged. #}
     {% if api.requires_package(('google', 'iam', 'v1')) or opts.add_iam_methods or api.has_iam_mixin %}
-        'grpc-google-iam-v1 >= 0.12.4, < 1.0.0dev',
+    "grpc-google-iam-v1 >= 0.12.4, < 1.0.0dev",
     {% endif %}
-    {% if api.requires_package(('google', 'cloud', 'documentai', 'v1')) %}
-        'google-cloud-documentai >= 1.2.1, < 3.0.0dev',
+    {% if api.requires_package(('google', 'cloud', 'documentai', 'v1')) and api.naming.warehouse_package_name != 'google-cloud-documentai' %}
```

Comment:
Could you file an issue and refer to it in a TODO to have a more elegant way of handling these cases? It seems wrong to hard-code packages and versions in the template (brittle, technical debt).

Reply:
I've moved these cases to a separate file and extracted the common code.